pyflakes/test/test_type_annotations.py

```diff
@@ -231,6 +231,14 @@ def f(a, b):
             return a + b
         """)
 
+    def test_typeCommentsStarArgs(self):
+        self.flakes("""
+        from mod import A, B, C, D
+        def f(a, *b, **c):
+            # type: (A, *B, **C) -> D
```

Comment:
That's a valid type sig? I didn't know that.

Reply:
most of the time I've seen it `C` will end up being `Any`

as in:
```python
def f(*args: str, **kwargs: Any) -> None:
   ...
```